.travis.yml

```diff
@@ -58,6 +58,7 @@ env:
         - CONF="--enable-schedule-scalable"
         - CONF="--enable-dpdk-zero-copy"
         - CONF="--disable-static-applications"
+        - CONF="--enable-pcapng-support"
```

Comment:
I'm thinking about limiting variety of build options here. Not in your PR, but there are just way tooo many

Reply:
Cuinit caught actua; bugs in the original implementation, i'd like to keep that

Reply:
@apalos I'm not arguing against this check. I'm thinking if we can lower the amount of different configurations that we check. Like having just big and little configs.